mariadb_skysql/README.md

```diff
@@ -0,0 +1,62 @@
+# Agent Check: MariaDB SkySQL
+
+## Overview
+
+This check monitors [MariaDB SkySQL][1]. SkySQL is the only database-as-a-service that combines the ease and innovation of MariaDB with the limitless scalability, high performance and availability, and the security of the cloud.. Build faster, deploy easily, and manage all of your databases from a single pane via DataDog.  Designed for a hybrid and multi-cloud future, built on Kubernetes, and engineered for mission-critical transactional and analytical deployments, it’s the database-as-a-service you’ve been waiting for.  Ensure your data and cloud scale to your innovating data utilization needs, in an efficient, economical manner.
```

Comment:
The overview should not include marketing copy. The overview should include: 

1. What does Syksql do in 1-2 sentences? 
2. What will customers get from this integration?
3. What types of data sent in and the value of the data (how does this data help them)?

Reply:
@rtrieu and @emarsha94. Thanks for the suggestions. We will update the overview accordingly.